LDAP-Auth/Config/configPage.html

```diff
@@ -141,36 +159,98 @@ <h2 class="sectionTitle">LDAP Settings:</h2>
-                    config.LdapUsernameAttribute = LdapConfigurationPage.txtLdapUsernameAttribute.value;
-                    config.LdapSearchFilter = LdapConfigurationPage.txtLdapSearchFilter.value;
-                    config.LdapAdminFilter = LdapConfigurationPage.txtLdapAdminFilter.value;
-                    config.LdapBindUser = LdapConfigurationPage.txtLdapBindUser.value;
-                    config.LdapBindPassword = LdapConfigurationPage.txtLdapBindPassword.value;
-                    config.CreateUsersFromLdap = LdapConfigurationPage.chkEnableUserCreation.checked;
-                    config.EnableCaseInsensitiveUsername = LdapConfigurationPage.chkEnableCaseInsensitiveUsername.checked;
-                    window.ApiClient.updatePluginConfiguration(LdapConfigurationPage.pluginUniqueId, config).then(Dashboard.processPluginConfigurationUpdateResult);
+                        html += '</div>';
+                        LdapConfigurationPage.folderAccessList.innerHTML = html;
+                        /* Only show these options if Jellyfin user creation is enabled on successful LDAP authentication */
+                        updateFolderListVisibility();
+                        LdapConfigurationPage.chkEnableUserCreation.addEventListener('change', updateFolderListVisibility);
+                        /* Set up event handlers for tracking folder enabling/disabling */
+                        Array.prototype.forEach.call(document.querySelectorAll('#folderList input'), (folder) => {
+                          folder.addEventListener('change', (event) => {
+                            const folders = document.querySelectorAll('#folderList input');
+                            let count = 0;
+                            Array.prototype.forEach.call(folders, folder => folder.checked && count++);
+                            LdapConfigurationPage.chkEnableAllFolders.checked = count == folders.length;
+                          });
+                        });
+                        LdapConfigurationPage.chkEnableAllFolders.addEventListener('change', (event) => {
+                          Array.prototype.forEach.call(document.querySelectorAll('#folderList input'), (folder) => {
+                            folder.checked = event.currentTarget.checked;
+                          });
+                        });
+                      });
+                    }
                 });
 
-                // Disable default form submission
-                return false;
-            });
+                document.querySelector(".esqConfigurationForm").addEventListener("submit", function(e){
+                    e.preventDefault();
+                    Dashboard.showLoadingMsg();
+
+                    window.ApiClient.getPluginConfiguration(LdapConfigurationPage.pluginUniqueId).then(function (config) {
+                        config.LDAPServer = LdapConfigurationPage.txtLdapServer.value;
```

Comment:
```suggestion
                        config.LdapServer = LdapConfigurationPage.txtLdapServer.value;
```

Unsure how this used to work... but without this change I can't save the ldap server